platform/linux-generic/pktio/dpdk.c

```diff
@@ -1120,21 +1124,34 @@ static int dpdk_pktio_init(void)
 		return -1;
 	}
 
-	mem_str_len = snprintf(NULL, 0, "%d", DPDK_MEMORY_MB);
+	mem_str_len = snprintf(NULL, 0, "%d,", DPDK_MEMORY_MB);
+
+#if defined(__arm__) && !defined(__aarch64__)
+	numa_nodes = 1;
+# else
+	numa_nodes = numa_num_configured_nodes();
```

Comment:
That looks like a hack. Instead of cross build libnuma you ifdefed call here. If you want make code working without libnuma let ./configure to test if libnuma exist and set variable which you will use here.

Reply:
The problem here is not cross-building of libnuma, but rather libnuma not supporting 32-bit ARM platforms (well, kernel not supporting `migrate_pages` syscall on that platform).

Reply:
thanks Dmirty. Then maybe to put this call under odp arch specific files?

Reply:
I'm ok with that.

Reply:
@lumag, @muvarov Isn't the real question whether `#include <numa.h>` is available to avoid compilation failures? As long as `numa_num_configured_nodes()` returns 1 for non-numa environments this should be good.

Reply:
@Bill-Fischofer-Linaro That was the reason I added the ifdef around the #include. However, at least on Travis the numa.h seems to be available even on arm32 cross compilation.